DirectProgramming/DPC++/OpenCLInterop/README.md

```diff
@@ -0,0 +1,77 @@
+# DPC++ OpenCL&trade; Interoperability Example
+
+The examples here demonstrate how DPC++ can interact with OpenCL&trade;. This enables programmers to incrementally migrate from
+OpenCL to DPC++. Two usage scenarios are shown. First is a DPC++ program that compiles and runs an OpenCL kernel. The second program converts OpenCL objects to DPC++.
+
+For more information on migrating from OpenCL to DPC++, see [Migrating OpenCL Designs to DPC++](https://software.intel.com/content/www/us/en/develop/articles/migrating-opencl-designs-to-dpcpp.html).
+
+| Optimized for                       | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04, 20; Windows 10
```

Comment:
Why does this sample show as working on windows, as I am not finding instructions in readme and json is linux only?

Reply:
That's just an artifact of copying. Although the code itself should run on windows as well. I've removed windows 10 from README